packages/search-ui/src/__tests__/A11yNotifications.ssr.test.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @jest-environment node
```

Comment:
This is awesome @constancecchen, I didn't know that you could do this.

Reply:
Haha! Yeah, I had to deal with this when working on my previous React library to prevent React portals from crashing SSR apps. Here's Jest's full docs for context if that's helpful at all:

https://jestjs.io/docs/en/configuration#testenvironment-string